src/SnapshotStack.php

```diff
@@ -30,4 +30,12 @@ public function getSnapshots(): array
 		return $this->snapshots;
 	}
 
+	/**
+	 * @return int
+	 */
+	public function getNumberOfSnapshots(): int
```

Comment:
Why is it needed? I prefer `count(this->snaphots)`

Reply:
I'ts the same. Just because I used on two place - DRY

Reply:
Just replace it as I said, thank you. I don't wanna introduce more methods if it's just `count($var)`.